lips/dataset/pneumaticWheelDataSet.py

```diff
@@ -631,351 +182,25 @@ def extract_data(self, concat: bool=True) -> tuple:
             extract_y = np.concatenate(extract_y, axis=1)
         return extract_x, extract_y
 
-    def __getitem__(self, item:int)->tuple:
+    def __eq__(self, other)->bool:
+        if sorted(self.data.keys())!=sorted(other.data.keys()):
+            return False
+        for dataAttribName in self.data.keys():
+            try:
+                np.testing.assert_almost_equal(self.data[dataAttribName],other.data[dataAttribName])
+            except AssertionError:
+                return False
+        return True
+
+    def __getitem__(self, index:int)->tuple:
         currentInput = {inputName:self.data[inputName][item] for inputName in self._attr_x}
```

Comment:
I think the `item` in `currentInput = {inputName:self.data[inputName][item]` should be `index`